src/components/AccountTransactionsPanel.vue

```diff
@@ -43,6 +43,8 @@ const pageIndex = ref(1)
 
 const limit = computed(() => isDesktop() ? 10 : 3)
 
+fetchAccountTransactionsCount(route.params.id, selectedTxType.value.typeQuery)
```

Comment:
please use await useAsyncData similarly to https://github.com/aeternity/aescan/pull/273/files. Otherwise, the backend will not wait until this request is finished and return the response prematurely.

Reply:
fixed